### PR TITLE
chore: changesets

### PR DESCRIPTION
**Pull request by osis**
**Short description of work done**
Adds changesets to help with release cycles.

### PR Checklist

- [x] I have run linter locally
- [x] I have run unit and integration tests locally
- [x] Update configuration the newest version (readme and const)
- [x] Rebased to master branch / merged master

### CODE REVIEW

File: .changeset/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Changesets
+
```
Comment (RayzorFlash) [inline .changeset/README.md:2]: Do we need this readme?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+      - name: Install Dependencies
+        run: yarn
+
+      - name: Create Release Pull Request or Publish to npm
+        id: changesets
+        uses: changesets/action@v1
+        with:
+          # This expects you to have a script called release which does a build for your packages and calls changeset publish
+          publish: echo "moo"
```
Comment (RayzorFlash) [inline .github/workflows/release.yml:31]: 🐮 ?

**Review by RayzorFlash — APPROVED** [review]
aside from the 🐮 , LGTM

File: .changeset/README.md (hunk 1 above)
Comment (osis) [reply .changeset/README.md:2]: hmmmmm probably not

**Pull request merged**